docs/configuration.md

```diff
@@ -3853,6 +3853,11 @@ The section contains the following settings (available since v3.8.1):
   - will only apply the rewrite if the cumulative span of all visible
     (non-whitespace) tokens within the argument is between the two values
   - this rule is disabled if `fewerBracesMaxSpan == 0`
+- (since v3.8.4) `fewerBracesParensToo`
```

Comment:
```suggestion
- (since v3.8.4) `alsoParens`
```
wouldn't be enough? Since it's already `removeOptionalBraces.fewerBracesParensToo`

Reply:
will that not be confusing? it only works for fewer braces, not for any other potential case involving optional braces.